src/Sylius/Bundle/InventoryBundle/DependencyInjection/Configuration.php

```diff
@@ -87,7 +88,7 @@ private function addResourcesSection(ArrayNodeDefinition $node)
                                         ->scalarNode('model')->defaultValue(InventoryUnit::class)->cannotBeEmpty()->end()
                                         ->scalarNode('interface')->defaultValue(InventoryUnitInterface::class)->cannotBeEmpty()->end()
                                         ->scalarNode('controller')->defaultValue(InventoryUnitController::class)->cannotBeEmpty()->end()
-                                        ->scalarNode('repository')->cannotBeEmpty()->end()
+                                        ->scalarNode('repository')->defaultValue(InventoryUnitRepository::class)->cannotBeEmpty()->end()
```

Comment:
This value shouldn't be added here, because it would couple implementation with ORM. It should be added in separated section under `Resource/config/driver/doctrine/` in the `orm.xml` file like [here](https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/ProductBundle/Resources/config/driver/doctrine/orm.xml). It is possible, that you will have to update [SyliusInventoryExtension](https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/InventoryBundle/DependencyInjection/SyliusInventoryExtension.php) too.